src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2932,6 +2933,22 @@ private uint getJitFlags(ref CORJIT_FLAGS flags, uint sizeInBytes)
                 {
                     throw new RequiresRuntimeJitException("ReadyToRun: Methods with UnmanagedCallersOnlyAttribute not implemented");
                 }
```

Comment:
The `#endif` should be here instead.

Reply:
Do we want this same check in CoreRT? I thought the restrictions were only for CoreCLR.

Reply:
The x86 restrictions are for CoreCLR only. The rest should be universal, for all runtimes.